suite-native/transactions/src/components/TransactionDetail/TransactionDetailInputsSheet.tsx

```diff
@@ -0,0 +1,115 @@
+            title="Inpust & outputs"
+            iconName="swap"
+            transactionId={transaction.txid}
+        >
+            <Box
+                flexDirection="row"
+                justifyContent="space-between"
+                marginHorizontal="small"
+                marginBottom="medium"
+            >
+                <Box flex={1} flexDirection="row" alignItems="center" paddingLeft="small">
+                    <Text variant="hint" color="gray600">
+                        Inputs · {txInputs.length}
+                    </Text>
+                    <Box marginLeft="small">
+                        <Icon name="receiveAlt" color="gray600" size="medium" />
+                    </Box>
+                </Box>
+
+                <Box flex={1} flexDirection="row" alignItems="center" paddingLeft="large">
+                    <Text variant="hint" color="gray600">
+                        Outputs · {txOutputs.length}
+                    </Text>
+                    <Box marginLeft="small">
+                        <Icon name="sendAlt" color="gray600" size="medium" />
+                    </Box>
+                </Box>
+            </Box>
+            <VStack marginBottom="medium">
+                <Card>
+                    <Box flexDirection="row" justifyContent="space-between">
+                        <Box>
+                            {txInputs.map((input, index) => (
+                                <TransactionAddressAmount
+                                    key={input.hex || index}
+                                    address={formatInputAddress(input.addresses![0])}
+                                    amount={CryptoAmountFormatter.format(
+                                        formatNetworkAmount(input.value ?? '0', transaction.symbol),
+                                        {
+                                            symbol: transaction.symbol,
+                                        },
+                                    )}
+                                />
+                            ))}
+                        </Box>
+                        <Icon name="circleRight" color="gray500" size="medium" />
+                        <Box>
+                            {txOutputs.map((output, index) => (
+                                <TransactionAddressAmount
+                                    key={output.hex || index}
+                                    address={formatInputAddress(output.addresses![0])}
+                                    amount={CryptoAmountFormatter.format(
+                                        formatNetworkAmount(
+                                            output.value ?? '0',
+                                            transaction.symbol,
+                                        ),
+                                        {
+                                            symbol: transaction.symbol,
+                                        },
+                                    )}
```

Comment:
Have TransactionAddressAmount accept amount and format it in the component itself

Reply:
That would require also sending an extra prop of `currency symbol`. Do you think that it's needed? Why is this solution wrong?

Reply:
It's just to make this a little bit cleaner as it's hard to read imo. You can definitely make a constant before render from this: 
```
formatNetworkAmount(
                                            output.value ?? '0',
                                            transaction.symbol,
                                        ),
                                        {
                                            symbol: transaction.symbol,
                                        },
```

Reply:
ideally I would let the function handle this though. Probably not going to be such a performance issue to pass a variable to the function 😅